viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move
+
+
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie_dict= {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict

```

Comment:
Great logic here! A few small notes:
- The `else` is unneeded because the `if` will return early if triggered.
- Consider making the dictionary all at once:
```
new_movie = {
        "title": title,
        "genre": genre,
        "rating": rating
}
```
- Teeny tiny nitpick: when checking if something is None, we want to say `title is None` instead of using `==`. For pretty much everything else we use `==`, but `None` is special. See here for [details](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none) if you're interested.